modules/mono/utils/naming_utils.cpp

```diff
@@ -60,8 +60,6 @@ const HashMap<String, String> pascal_case_part_overrides = _create_hashmap_from_
 		{ "AA", "AA" }, // Anti Aliasing
 		{ "AO", "AO" }, // Ambient Occlusion
 		{ "FILENAME", "FileName" },
-		{ "FADEIN", "FadeIn" },
-		{ "FADEOUT", "FadeOut" },
```

Comment:
@mhilbrunner The CSharp test has an error, so I looked into it, and it seems that your point is partially incorrect.

It seems that the method `get_fade_in_time` has already been converted to the method `GetFadeInTime` by `naming_utils.cpp`. However, since the other part naming in CPP are consistent with `fade_in`, so this "fadein -> FadeIn" conversion seems to be currently only used for NodeOneShot and it looks entirely like a hack.

In summary:

- I added `set_fade_in_time` here, which is converted correctly to `SetFadeInTime` without `naming_utils.cpp` in CSharp.

- Also, I remain the deprecated method `set_fadein_time` for compatibility. However, `naming_utils.cpp` will convert it to `SetFadeInTime`, which will result in duplicated declarations and errors, so we need to remove it (and again, this is only used for NodeOneShot, so removing it here should not be a problem). One problem that occurs as a side effect of that is the addition of a new method `SetFadeinTime` to CSharp.

Reply:
Since this is just a rename and doesn't break compat, I think it's fine for the C# implementation to call the new `get_fade_in_time`.

As for the new deprecated `SetFadeinTime` method, it'd be nice to avoid generating it. We can probably skip it like we do with [some classes](https://github.com/godotengine/godot/blob/964a5353dbf5fc0dca4c4b7b9cfa5631dd5ab568/modules/mono/editor/bindings_generator.cpp#L117-L118) but this can probably be done in a separate PR as long as it gets merged before the new API ships. But I don't think it would be a big deal even if we end up shipping the new deprecated API.